docs/snippets/GPU_Metric1.cpp

```diff
@@ -1,25 +1,23 @@
-#include <ie_core.hpp>
+#include <openvino/runtime/core.hpp>
+#include <openvino/runtime/intel_gpu/properties.hpp>
 
 int main() {
-using namespace InferenceEngine;
 //! [part1]
-InferenceEngine::Core core;
-CNNNetwork cnnNetwork = core.ReadNetwork("network.xml");
+ov::Core core;
+std::shared_ptr<ov::Model> network = core.read_model("sample.xml");
 uint32_t n_streams = 2;
 int64_t available_device_mem_size = 3221225472;
+std::map<std::string, ov::Any> options = {{ov::hint::model_ptr.name(), network}}; // Required. Set the address of the target network. If this is not set, the MAX_BATCH_SIZE returns 1.
+options.insert(std::make_pair(ov::streams::num.name(), n_streams)); // Optional. Set only when you want to estimate max batch size for a specific throughtput streams. Default is 1 or throughtput streams set by SetConfig.
+options.insert(std::make_pair(ov::intel_gpu::hint::available_device_mem.name(), available_device_mem_size)); // Optional. Set only when you want to limit the available device mem size.
```

Comment:
why not `options[ov::intel_gpu::hint::available_device_mem] = available_device_mem_size` ?

Reply:
`options.insert(ov::intel_gpu::hint::available_device_mem(available_device_mem_size));` is also supported

Reply:
```c++
ov::AnyMap properties = {
    ov::hint::model_ptr(model), 
    ov::streams::num(n_streams), 
    ov::intel_gpu::hint::available_device_mem(available_device_mem_size)
};
```

Reply:
@ilya-lavrenov, in current implementation we can't use ov::Property directly as a key
@apankratovantonp, I applied your suggestion, thanks